server/routes/api/chart.py

```diff
@@ -213,48 +210,74 @@ def choropleth_data(dcid):
     API Returns:
         Dictionary with
             key as stat var
-            value as object with date,dictionary of place: value, number of data points, exploreUrl, list of sources
+            value as object with date,dictionary of place: value, number of
+            data points, exploreUrl, list of sources
     """
-    all_stat_vars, choropleth_configs = get_choropleth_sv()
-    geos, _ = get_choropleth_places(dcid)
-    if not all_stat_vars or not geos:
+    configs = get_choropleth_configs()
+    stat_vars, denoms = get_stat_vars(configs)
+    display_dcid, display_level = get_choropleth_display_level(dcid)
+    geos = dc_service.get_places_in([display_dcid],
```

Comment:
Do you see this approach faster than before?

Reply:
I actually haven't noticed it being faster